sdk/typescript/graphql/client.ts

```diff
@@ -1,6 +1,27 @@
 import * as opentelemetry from "@opentelemetry/api"
 import { GraphQLClient } from "graphql-request"
 
+const createFetchWithTimeout =
+  (timeout: number) => async (input: RequestInfo | URL, init?: RequestInit) => {
+    if (init?.signal) {
+      throw new Error(
+        "it looks like graphql-request started using AbortSignal on its own. Please check graphql-request's recent updates",
```

Comment:
:eyes: This is a *very* strange error to send to the user - this is an internal error, we should mark it as such, I don't think the user can do anything about it.

The suggestion for what to do can be a code comment IMO.

Reply:
It's good as is, but the error message that was being thrown previously would have been better suited as a code comment imo.